### PR TITLE
[FIX] fieldservice_purchase: Access rights

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Review by nelsonramirezs — APPROVED** [review]
LGTM

**Review by dreispt — APPROVED** [review]

File: fieldservice_purchase/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_purchase_order_fsm_user,purchase.order.fsm.user,model_purchase_order,fieldservice.group_fsm_user,1,0,0,0
```
Comment (dreispt) [inline fieldservice_purchase/security/ir.model.access.csv]:
```suggestion
access_purchase_order_fsm_user,purchase.order.fsm.user,purchase.model_purchase_order,fieldservice.group_fsm_user,1,0,0,0
```

**Review by dreispt — CHANGES_REQUESTED** [review]

**Pull request merged**